src/modules/react-reconciler/Atlas.tsx

```diff
@@ -445,6 +445,11 @@ export const Atlas: React.FC<
 
   const { height: _, width: __, ...canvasProps } = restProps;
   const widthClassName = useClassname([restProps.width, restProps.height]);
+  let isInteractive = true;
+  // if we have a render preset and that render preset sets interactive to false, then... disable it
+  if (renderPreset && Array.isArray(renderPreset) && renderPreset.length > 1 && (renderPreset[1] as any).interactive === false) {
+    isInteractive = false;
+  }
```

Comment:
is this a logical assumption, by default it's interactive, but if someone explicitly sets the presets to disable the interactivity, then it isn't